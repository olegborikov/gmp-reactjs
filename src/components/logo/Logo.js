import React from "react";
import classes from './Logo.module.css';

function Logo() {
  return (
    <div className={classes.logo}>
      <strong>netflix</strong>roulette
    </div>
  )
}

export default Logo
