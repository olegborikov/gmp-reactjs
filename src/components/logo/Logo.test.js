import React from "react";
import {render} from "@testing-library/react";
import Logo from "./Logo";

it("snapshot test", () => {
  const {asFragment} = render(<Logo/>)
  expect(asFragment()).toMatchSnapshot()
})
