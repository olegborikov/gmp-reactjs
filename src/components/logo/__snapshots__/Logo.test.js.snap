// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<DocumentFragment>
  <div
    class="logo"
  >
    <strong>
      netflix
    </strong>
    roulette
  </div>
</DocumentFragment>
`;
